tools/Packer/ConfigModel.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Packer
+{
+    public class ConfigModel
+    {
+        public string CloneDir;
+        public string CloneUrl;
+        public string Version;
+        public List<string> PathsToRemove = new List<string>();
+        public List<GitDependency> GitDependencies = new List<GitDependency>();
```

Comment:
It's kind of an odd structure that there's a top level "git dependency" and then nested ones. Why not put it all in a single list and require that there be at least one item in it?

Reply:
Refactored this to what you suggest